src/Toml/Bi/Combinators.hs

```diff
@@ -298,3 +302,15 @@ list codec key = Codec
   where
     nonEmptyCodec :: TomlCodec (NonEmpty a)
     nonEmptyCodec = nonEmpty codec key
+
+set :: forall a . Ord a => TomlCodec a -> Key -> TomlCodec (Set a)
+set codec key = dimap (NE.fromList . S.toList) (S.fromList . NE.toList) nonEmptyCodec
```

Comment:
You can use `list` codec instead of `nonEmpty` to avoid manual conversion from/to `NonEmpty`. Also, `fromList` is an unsafe function and can crash in runtime.

Reply:
You mean I shouldn't use the S.fromList function or was that for NE?

Ah I missed seeing that I could use list codec.

Reply:
You shouldn't use `NE.fromList`.